spec/features/course/achievement_management_spec.rb

```diff
@@ -108,6 +108,158 @@
           expect(achievement.reload.description).to eq(new_description)
         end
       end
+
+      context 'achievement achievement conditions' do
+        let(:achievement) { create(:course_achievement, course: course) }
+        it { is_expected.to have_selector(:link_or_button,
+                                          'course.condition.achievements.new.header') }
+        context 'creating an achievement condition' do
+          before do
+            click_link 'course.condition.achievements.new.header'
+            expect(current_path).to eq(new_course_achievement_condition_achievement_path(course, achievement))
+            find('#condition_achievement_achievement_id').
+              find(:css, "option[value='#{achievement.id}']").
+              select_option
+            click_button 'helpers.submit.condition_achievement.create'
+          end
+          it 'creates an achievement condition correctly' do
+            expect(current_path).to eq edit_course_achievement_path(course, achievement)
+            expect(page).to have_selector('tr.condition > td:nth-child(2)', text: achievement.title)
+          end
+        end
+        context 'editing and deleting' do
+          let!(:achievement_which_is_condition) { create(:course_achievement, course: course) }
+          let(:achievement_condition) { create(:course_condition_achievement, course: course,
+                                               achievement: achievement_which_is_condition) }
+          let!(:some_other_achievement) { create(:course_achievement, course: course) }
+          let!(:achievement) { create(:course_achievement, course: course,
+                                      conditions: [achievement_condition]) }
```

Comment:
Align the elements of a hash literal if they span more than one line.<br>Expression at 136, 76 should be on its own line.